### PR TITLE
abt policy updates

### PR DESCRIPTION
**Pull request by davidmu1**

### CODE REVIEW

**Comment by davidmu1** [comment]
Sorry @VesS0 , mistakenly included you in review.

File: api-reference/beta/resources/activitybasedtimeoutpolicy.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "activityBasedTimeoutPolicy resource type"
+description: "Specifies a shared activity-based timeout policy."
+localization_priority: Normal
+author: davidmu1
+doc_type: resourcePageType
+ms.prod: "microsoft-identity-platform"
+---
+
+# activityBasedTimeoutPolicy resource type
```
Comment (dkershaw10) [inline api-reference/beta/resources/activitybasedtimeoutpolicy.md]: @davidmu1 Strictly this is NOT a resource.  It's a property in the `policy` resource that requires a lot more description.  I'm not sure how we want to name it.  It is related to a resource, but it's not a resource type.

Suggest - policy definition for activityBasedTimeoutPolicy type 

Need to apply the same to tokenLifetimePolicy

File: api-reference/beta/resources/policy.md [hunk 2]
```diff
@@ -1,42 +1,44 @@
...
-|[Assign policy](../api/policy-assign.md)|None|Assign a policy to an application, service principal.|
+|[Assign policy](../api/policy-assign.md)|None|Assign a policy to an application or service principal.|
 |[List policies](../api/policy-list.md)|Policy collection|Get all policy objects in the organization.|
 |[List assigned policies](../api/policy-list-assigned.md)|Policy collection|Get all policy objects assigned to an application or service principal.|
 
 ##  Properties
 | Property	   | Type	|Description|
 |:---------------|:--------|:----------|
-|definition|String collection|The string version of the specific policy. See below. Required.|
+|definition|String|The string version of the specific policy. See below. Required.|
```
Comment (dkershaw10) [inline api-reference/beta/resources/policy.md]: I think this should say something like:
The policy definition is a JSON string. Each policy definition JSON is different per policy type.  Currently the following definitions are supported: [list the 2 names and link to the 2 definitions].

File: api-reference/beta/resources/policy.md [hunk 3]
```diff
@@ -1,42 +1,44 @@
...
 |displayName|String|A custom name for the policy. Required.|
-|IsOrganizationDefault|Boolean|If set to true, activates this policy. There can be many policies for the same policy type, but only one can be activated as the organization default. Optional, default value is false.|
-|type|String|Specifies the type of policy. Currently must be "TokenLifetimePolicy". Required.|
-|alternativeIdentifer|String| |
-|keyCredentials| [keyCredential](keycredential.md) collection| |
+|isOrganizationDefault|Boolean|If set to true, activates this policy. There can be many policies for the same policy type, but only one can be activated as the organization default. Optional, default value is false.|
+|type|String|Specifies the type of policy. Required.|
```
Comment (dkershaw10) [inline api-reference/beta/resources/policy.md]: You should list the 2 supported types.

File: api-reference/beta/resources/activitybasedtimeoutpolicy.md (hunk 1 above)
Comment (davidmu1) [reply api-reference/beta/resources/activitybasedtimeoutpolicy.md]: Renamed to "activityBasedTimeoutPolicy policy definition"

File: api-reference/beta/resources/policy.md (hunk 2 above)
Comment (davidmu1) [reply api-reference/beta/resources/policy.md]: Revised the description and added the links.

File: api-reference/beta/resources/policy.md (hunk 3 above)
Comment (davidmu1) [reply api-reference/beta/resources/policy.md]: Added "Supported types: **activityBasedTimeoutPolicy** and **tokenLifetimePolicy**."

**Comment by davidmu1** [comment]
#sign-off

File: api-reference/beta/api/policy-post.md [hunk 4]
```diff
@@ -44,18 +44,17 @@ The following table shows the properties that are required when you create a pol
...
 
 ## Example
-The following example creates a new token lifetime Policy. Notice the string definition parameter
-has escaped double quotes.
+The following example creates a new token lifetime policy. Notice that the string definition parameter has escaped double quotes.
 
-##### Request
-Here is an example of the request.
+### Request
+The following example shows the request.
```
Comment (Lauragra) [inline api-reference/beta/api/policy-post.md]: @davidmu1 , FYI, these files appear to be missing the HTML comments required for validation. While we're updating the topics, can we add the missing comments?

**Comment by davidmu1** [comment]
#sign-off

File: api-reference/beta/api/policy-post.md [hunk 5]
```diff
@@ -44,33 +44,44 @@ The following table shows the properties that are required when you create a pol
 |:---------------|:--------|:----------|
 |definition|String|The string version of the [policy](../resources/policy.md) object.|
 |displayName|String|A custom name for the policy.|
-|type|String|Specifies the type of policy. Currently must be "TokenLifetimePolicy"|
+|type|String|Specifies the type of [policy](../resources/policy.md).|
```
Comment (dkershaw10) [inline api-reference/beta/api/policy-post.md]: How do I know what are the valid values I can put here?
How do I know what the definitions look like?

There are no links to the definition descriptions...

File: api-reference/beta/api/policy-post.md [hunk 6]
```diff
@@ -44,33 +44,44 @@ The following table shows the properties that are required when you create a pol
 |:---------------|:--------|:----------|
 |definition|String|The string version of the [policy](../resources/policy.md) object.|
```
Comment (dkershaw10) [inline api-reference/beta/api/policy-post.md]: According to schema, this is a string **collection**, although why it's a collection is anyone's guess.
Missing any links to definition schemas...

File: api-reference/beta/resources/policy.md [hunk 7]
```diff
@@ -1,45 +1,49 @@
...
 ## Methods
 | Method       | Return Type  |Description|
 |:---------------|:--------|:----------|
-| [Get policy](../api/policy-get.md) |Policy|Read properties and relationships of user object.|
+|[Get policy](../api/policy-get.md)|Policy|Read properties and relationships of user object.|
 |[Create policy](../api/policy-post.md)|Policy|Create a new policy object.|
 |[Update policy](../api/policy-update.md)|None|Update policy object.|
 |[Delete policy](../api/policy-delete.md)|None|Delete policy object.|
-|[Assign policy](../api/policy-assign.md)|None|Assign a policy to an application, service principal.|
+|[Assign policy](../api/policy-assign.md)|None|Assign a policy to an application or service principal.|
```
Comment (dkershaw10) [inline api-reference/beta/resources/policy.md]: You are missing "Remove policy"

File: api-reference/beta/resources/policy.md [hunk 8]
```diff
@@ -1,45 +1,49 @@
...
-|definition|String collection|The string version of the specific policy. See below. Required.|
+|definition|String|The policy definition is a JSON string. Each policy definition JSON is different per policy type. Supported definitions: [activitybasedtimeoutpolicy](activitybasedtimeoutpolicy.md) and [tokenlifetimepolicy](tokenlifetimepolicy.md). Required.|
 |displayName|String|A custom name for the policy. Required.|
-|IsOrganizationDefault|Boolean|If set to true, activates this policy. There can be many policies for the same policy type, but only one can be activated as the organization default. Optional, default value is false.|
-|type|String|Specifies the type of policy. Currently must be "TokenLifetimePolicy". Required.|
-|alternativeIdentifer|String| |
-|keyCredentials| [keyCredential](keycredential.md) collection| |
+|isOrganizationDefault|Boolean|If set to true, activates this policy. There can be many policies for the same policy type, but only one can be activated as the organization default. Optional, default value is false.|
+|type|String|Specifies the type of policy. Supported types: **activityBasedTimeoutPolicy** and **tokenLifetimePolicy**. Required.|
```
Comment (dkershaw10) [inline api-reference/beta/resources/policy.md]: Please Pascal-case the type values.

File: api-reference/beta/resources/policy.md [hunk 9]
```diff
@@ -1,45 +1,49 @@
...
-|[Assign policy](../api/policy-assign.md)|None|Assign a policy to an application, service principal.|
+|[Assign policy](../api/policy-assign.md)|None|Assign a policy to an application or service principal.|
 |[List policies](../api/policy-list.md)|Policy collection|Get all policy objects in the organization.|
 |[List assigned policies](../api/policy-list-assigned.md)|Policy collection|Get all policy objects assigned to an application or service principal.|
 
 ##  Properties
 | Property	   | Type	|Description|
 |:---------------|:--------|:----------|
-|definition|String collection|The string version of the specific policy. See below. Required.|
+|definition|String|The policy definition is a JSON string. Each policy definition JSON is different per policy type. Supported definitions: [activitybasedtimeoutpolicy](activitybasedtimeoutpolicy.md) and [tokenlifetimepolicy](tokenlifetimepolicy.md). Required.|
```
Comment (dkershaw10) [inline api-reference/beta/resources/policy.md]: It's a string collection, that can only ever have one string in the collection.
Please make these Pascal-cased to match the type value names.

File: api-reference/beta/resources/tokenlifetimepolicy.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
+---
+title: "tokenLifetimePolicy policy definition"
+description: "Specifies the lifetimes of tokens issued for various purposes"
+localization_priority: Normal
+author: davidmu1
+doc_type: resourcePageType
+ms.prod: "microsoft-identity-platform"
+---
+
+# tokenLifetimePolicy policy definition
```
Comment (dkershaw10) [inline api-reference/beta/resources/tokenlifetimepolicy.md]: I think this should be Pascal-cased (unfortunately)

File: api-reference/beta/resources/activitybasedtimeoutpolicy.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
+---
+title: "activityBasedTimeoutPolicy policy definition"
+description: "Specifies a shared activity-based timeout policy."
+localization_priority: Normal
+author: davidmu1
+doc_type: resourcePageType
+ms.prod: "microsoft-identity-platform"
+---
+
+# activityBasedTimeoutPolicy policy definition
```
Comment (dkershaw10) [inline api-reference/beta/resources/activitybasedtimeoutpolicy.md]: I think this should be Pascal-cased (unfortunately)

**Pull request closed without merging**